R/post_process_plot.R

```diff
@@ -45,6 +45,11 @@ post_process_plot <- function(plot, is_sorted = TRUE, horizontal = TRUE,
 
   }
 
+  if (other & !is.null(threshold)) {
+    caption <-  paste0("'Other' contains all categories with a value less than or equal to", threshold, ".")
```

Comment:
Now you are missing a space between `"to"` and `threshold`. You need to explicitly add it but keep using `paste0` to not have a space in front of `"."`.